testsuite/cluster-test/src/cluster_swarm/configs/validator.yaml

```diff
@@ -66,7 +66,8 @@ failpoints:
   executor::vm_execute_chunk: 1%return
   executor::commit_chunk: 1%return
   executor::vm_execute_block: 1%return
-  #executor::commit_blocks: 0.01%return
+  executor::commit_blocks: 0.01%return
```

Comment:
should this be uncommented?

Reply:
i'm not sure.. @lightmark

Reply:
let's chain our @... 
This line was intentional based on some injection glitches. @zekun000 solved?

Reply:
the uncomment was accidental, left over from when i was playing around with it locally. will revert for now

Reply:
I think it caused the cluster test to fail because this panics the pod (with 1% maybe?)